CMakeLists.txt

```diff
@@ -2,24 +2,32 @@ cmake_minimum_required(VERSION 3.5.0)
 project(opw_kinematics VERSION 0.1.0 LANGUAGES CXX)
 
 find_package(Eigen3 REQUIRED)
+find_package(cmake_common_scripts REQUIRED)
+
+add_code_coverage_all_targets()
```

Comment:
This probably (maybe) overwrites your enable flags in the target_add_code_coverage?

Reply:
Fixed